src/main/java/com/backendoori/ootw/common/image/MiniOImageServiceImpl.java

```diff
@@ -19,6 +19,7 @@
 @RequiredArgsConstructor
 public class MiniOImageServiceImpl implements ImageService {
 
+    private static final int DURATION = 12;
     private final MinioClient minioClient;
```

Comment:
상수 부분은 띄어쓰는게 좋을 것 같아요

```suggestion
    private static final int DURATION = 12;
    
    private final MinioClient minioClient;
```